vuepress/docs/next/tutorials/ecr/ecr-how-microservices-connects-to-apps.md

```diff
@@ -1,66 +1,66 @@
-# ECR: how microservices get connected to Entando Apps
+# ECR: How Microservices get Connected to Entando Apps
 
 In this document we will describe some of the assumptions and details related to the process of connecting a microservice part of a bundle to an Entando App.
 
-## How a microservice part of a bundle get deployed with the ECR?
+## How a Microservice from a Bundle gets Deployed with the ECR?
 
 When a bundle containing a microservice is installed using the ECR, behind the scenes some actions take place.
 
 1. To begin, a check for a microservice with the same name is performed to verify if a new deployment is required for the microservice.
-2. If a microservice with the same name is not available in the namespace where the Entando App has been deployed, a new EntandoPlugin custom resource is created and
-deployed in the namespace using the details defined in the bundle.
+2. If a microservice with the same name is not available in the namespace where the Entando App has been deployed, a new EntandoPlugin custom resource is created and deployed in the namespace using the details defined in the bundle.
 3. At the same time, an EntandoAppPluginLink custom resource is deployed in the namespace in order to expose the microservice ingress path on the EntandoApp ingress.
-4. If both the EntandoPlugin custom resource and the EntandoAppPluginLink are deployed correctly, the APIs of the microservice will be available from the same domain of the 
-EntandoApp, making it possible reach those APIs from the EntandoApp using relative urls
+4. If both the EntandoPlugin custom resource and the EntandoAppPluginLink are deployed correctly, the APIs of the microservice will be available from the same domain of the EntandoApp, making it possible reach those APIs from the EntandoApp using relative urls
 
 This is the standard flow when no other micorservice with a given name is already available in the EntandoApp namespace.
 
-If such condition is not true and there's already microservice with the same name of the one described in the bundle, the ECR will connect directly the EntandoApp to
-the existing microservice by generating and deploying the required EntandoAppPluginLink as in the step 3 above. 
+If there is already microservice with the same name of the one described in the bundle, the ECR will connect directly the EntandoApp to
```

Comment:
Suggestion: If there is an existing microservice with the same name as the one described in the bundle, the ECR will connect the EntandoApp to
the existing microservice by generating and deploying the required EntandoAppPluginLink per step 3 above.

Reply:
@nshaw yeah please go ahead with the suggestion :+1: